src/main/resources/application.properties

```diff
@@ -12,63 +12,68 @@ quarkus.health.extensions.enabled=true
 
+%dev.quarkus.infinispan-client.server-list=localhost:11222
+%dev.quarkus.infinispan-client.near-cache-max-entries=1000
+%dev.quarkus.infinispan-client.auth-username=admin
+%dev.quarkus.infinispan-client.auth-password=admin
+
+%prod.quarkus.infinispan-client.server-list=${INFINISPAN_SERVICE_HOST}:${INFINISPAN_SERVICE_PORT}
+%prod.quarkus.infinispan-client.near-cache-max-entries=1000
+%prod.quarkus.infinispan-client.auth-realm=${INFINISPAN_REALM}
+%prod.quarkus.infinispan-client.auth-username=${INFINISPAN_USER}
+%prod.quarkus.infinispan-client.auth-password=${INFINISPAN_PASSWORD}
+
+## TrustStore retrieved from the java-truststore configmap, managed by cert-utils-operator
+%prod.quarkus.infinispan-client.trust-store=/deployments/truststore.jks
+%prod.quarkus.infinispan-client.trust-store-password=jkspassword
+#10 seconds between leaderboard updates
+
 ##OpenShift resource configuraton
-quarkus.kubernetes-client.trust-certs=true
-quarkus.kubernetes.deployment-target=openshift
-quarkus.openshift.expose=true
+#quarkus.kubernetes-client.trust-certs=true
+#quarkus.kubernetes.deployment-target=openshift
+#quarkus.openshift.expose=true
 #quarkus.container-image.group=petbattle
 #quarkus.container-image.name=tournamentsvc
 #quarkus.container-image.tag=latest
 #quarkus.openshift.name=tournamentsvc
-quarkus.kubernetes-config.enabled=true
-quarkus.kubernetes-config.secrets.enabled=true
-quarkus.kubernetes-config.fail-on-missing-config=true
-quarkus.openshift.labels.component=petbattle
+#quarkus.kubernetes-config.enabled=true
+#quarkus.kubernetes-config.secrets.enabled=true
+#quarkus.kubernetes-config.fail-on-missing-config=true
+#quarkus.openshift.labels.component=petbattle
 
 ##Configmaps and Secrets
-quarkus.openshift.secret-volumes.pbsecretcache.secret-name=tournamentcache-cert-secret
-quarkus.openshift.mounts.pbsecretcache.path=/tmp/secrets/cache
-quarkus.openshift.secret-volumes.pbsecretkeycloak.secret-name=keycloak-client-secret-pbclient
-quarkus.openshift.mounts.pbsecretkeycloak.path=/tmp/secrets/keycloak
-quarkus.openshift.config-map-volumes.javatruststore.config-map-name=java-truststore
-quarkus.openshift.mounts.javatruststore.path=/tmp/config/
+#quarkus.openshift.secret-volumes.pbsecretcache.secret-name=tournamentcache-cert-secret
+#quarkus.openshift.mounts.pbsecretcache.path=/tmp/secrets/cache
+#quarkus.openshift.secret-volumes.pbsecretkeycloak.secret-name=keycloak-client-secret-pbclient
+#quarkus.openshift.mounts.pbsecretkeycloak.path=/tmp/secrets/keycloak
+#quarkus.openshift.config-map-volumes.javatruststore.config-map-name=java-truststore
+#quarkus.openshift.mounts.javatruststore.path=/tmp/config/
 
 ##Base Image config
 #quarkus.s2i.base-jvm-image=registry.access.redhat.com/openjdk/openjdk-11-rhel7
 quarkus.openshift.env-vars.SCRIPT_DEBUG.value=false
 quarkus.openshift.env-vars.JAVA_OPTS.value=-Djavax.net.ssl.trustStore=/tmp/config/truststore.jks -Djavax.net.ssl.trustStorePassword=jkspassword
```

Comment:
This seems to be wrong location, the app.props says %prod.quarkus.infinispan-client.trust-store=/deployments/truststore.jks but the JAVA_OPTS are -Djavax.net.ssl.trustStore=/tmp/config/truststore.jks

Reply:
yeah .. this is me being a bit lazy .. i need to test this works again  and adjust - hence why i commented them out and not removed them .. i was really only testing the helm chart from a built image in quay.io for now .. however

the ci/cd process looks like this:

# local dev loop
mvn quarkus:dev  <--- local development
mvn package -Dquarkus.kubernetes.deploy=true  <--- this is all of the kube/ocp extension stuff above that i have not tested
make podman-push  <--- this builds the image using Dockerfile and does a mvn package and pushes to quay.io

# ci
tekton or jenkinsfile   <--- pipeline to build and test the image 

# cd
helm - then takes the build image and deploys the app using approriate %profile in application.properties. this can be from the CLI or using argocd of course